src/ffi/datetime.rs

```diff
@@ -16,6 +16,11 @@ use std::os::raw::{c_char, c_int, c_uchar};
 use std::ptr;
 use std::sync::Once;
 
+// A note regarding cpyext support:
```

Comment:
Should this be an actual docstring?

Reply:
I think it would be good to expose this document after adding some more explaination about `PyDateTime_CAPI` itself